calicoctl/commands/node/diags.go

```diff
@@ -155,7 +160,45 @@ such as transfer.sh using curl or similar.  For example:
 
 }
 
-// writeDiags executes the dignostic commans and outputs the result in the file
+// getHostedLogs will attempt to grab logs for any "calico" named containers for hosted installs.
+func getHostedLogs(logDir string) {
+	os.Mkdir(logDir, os.ModeDir)
+
+	// Get a list of Calico containers running on this Node.
+	var stderr bytes.Buffer
+	cmd := exec.Command("docker", "ps", "-a", "--filter", "\"name=calico\"", "--format", "{{.Names}}")
```

Comment:
Does this `--filter name=calico` actually work when Calico is deployed as a DaemonSet? I would think the name would be something like `k8s_calico-node-asdfasfdksdflahsdfhsdf`

Reply:
That can be a partial match and will return any containers with "calico" in them, so it will match, I tested this on a Vagrant install and got the expected output.